crates/bevy_render/src/render_resource/buffer_vec.rs

```diff
@@ -6,6 +6,18 @@ use bevy_core::{cast_slice, Pod};
 use copyless::VecHelper;
 use wgpu::BufferUsages;
 
+/// A structure for storing raw bytes that have already been properly formatted
+/// for the [`RenderDevice`](crate::renderer::RenderDevice).
+///
+/// "Properly formatted" means data that is [`#[repr(C)]`](https://doc.rust-lang.org/reference/type-layout.html#the-c-representation), [`Pod`]
+/// and `Zeroable`. Unlike the [`DynamicUniformVec`](crate::render_resource::DynamicUniformVec),
```

Comment:
Note that since we now use `encase`, the helper types were renamed to `UniformBuffer`, `DynamicUniformBuffer`, `StorageBuffer`, and `DynamicStorageBuffer`. I think I have made the necessary changes in the suggestion below.